_i18n/it/get-started/faq.md

```diff
@@ -1,136 +1,142 @@
+No, Monero non ha un limite sulla dimensione dei blocchi. Al contrario, la dimensione dei blocchi può aumentare o diminuire nel tempo a seconda del numero di transazioni da validare. Esiste un limite sulla velocità di crescita della dimensione del blocco al fine di evitare tassi di crescita eccessivi.
 </div>
 
 </div>
 
 <div class="tab">
     <input id="tab-nine" type="checkbox" name="tabs" class="accordion">
-    <label for="tab-nine" class="accordion">What is a blockchain?</label>
+    <label for="tab-nine" class="accordion">Cos'è la blockchain?</label>
     
 <div class="tab-content" markdown="1">
 
-A blockchain is a system that stores a copy of all transaction history on the Monero network. Every two minutes, a new block with the latest transaction information is added to the blockchain. This chain allows the network to verify the amount of money accounts have and make it resilient to attacks and centralization attempts.
+La "blockchain" (catena di blocchi) è un sistema grazie al quale è possibile mantenere una copia di tutta la storia delle transazioni sulla rete Monero. Ogni due minuti, viene aggiunto alla blockchain un nuovo blocco contenente le informazioni sulle ultime transazioni. Questa catena consente alla rete di verificare la quantità di denaro in circolazione e di essere resiliente ad attacchi e tentativi di centralizzazione.
 </div>
 
 </div>
 
 <div class="tab">
     <input id="tab-ten" type="checkbox" name="tabs" class="accordion">
-    <label for="tab-ten" class="accordion">What is Kovri?</label>
+    <label for="tab-ten" class="accordion">Cos'è Kovri?</label>
     
 <div class="tab-content" markdown="1">
 
-Kovri is an I2P router written in C++. I2P is a hidden network like Tor with several technical differences. Kovri is an independent project of Monero, but it will work with Monero and several other projects. Kovri hides the transaction broadcast, so other nodes do not know who created transactions. In adversarial conditions, Kovri can be used to hide all Monero traffic through I2P, which would prevent people from knowing Monero is being used. Kovri is currently in alpha, and it is not yet fully integrated in Monero. Learn more about Kovri at the [project website.](https://getkovri.org)
+Kovri è un router I2P sviluppato in C++. I2P è un network anonimo, come Tor, ma presenta parecchie differenze tecnologiche. Kovri è un progetto indipendente da Monero, ma funzionerà con Monero così come con altri progetti. Kovri offusca la trasmissione delle transazioni in modo tale che gli altri nodi non possano risalire a chi ha iniziato la transazione. In scenari ostili, Kovri può essere usato per offuscare tutto il traffico Monero attraverso I2P, in modo tale che non possa essere dimostrato l'utilizzo di Monero. Kovri è al momento in versione alpha e non è ancora pienamente integrato con Monero. Maggiori informazioni su Kovri sul [sito di progetto.] (https://getkovri.org)
```

Comment:
This is markdown syntax, please remove the space between `[sito di progetto.]`and `(https://getkovri.org)` or the link won't work properly

Reply:
Yup!